Resources/Prototypes/Nyanotrasen/Catalog/default_shipyard_catalog.yml

```diff
@@ -60,3 +60,12 @@
   category: Medium
   group: Civilian
   shuttlePath: /Maps/Shuttles/ntv-pulse.yml
+
+- type: vessel
+  id: Ranger
+  name: NTSV Ranger
+  description: A middling expeditionary shuttle equipped for brief away missions involving at most six crew.
+  price: 45000
+  category: Medium
+  group: Civilian
```

Comment:
I guess all our current shuttles are 'civilian' but if there's a good opportunity to start sorting them this is it.

Reply:
Good idea. I'll take a look to see if they're prototype-driven and set up an "Expedition" group. Though, I'm not sure if this is visible to the player at all? What exactly does this represent?

Reply:
@Cheackraze

Reply:
They look currently unused...

Reply:
https://github.com/Nyanotrasen/Nyanotrasen/issues/1606

Reply:
sorry I somehow missed this, I'll take a look real quick

Reply:
the `group: civilian ` isnt presented to the player but the category is, in the drop-down selector.  originally the 'group: ' parameter was intended for console-specific sorts.